07-sampling.Rmd

```diff
@@ -73,128 +70,111 @@ percent_red_chr <- prop_red %>% percent(accuracy = 0.1)
 
 
-### Using the shovel 33 times {#student-shovels}
+### Thirty-three samples {#student-shovels}
 
-Each of our 33 groups of friends will do the following: 
+To understand why this work well, we repeat this activity many times as shown in Figure \@ref(fig:sampling-exercise-3b). Each time we do the following: 
 
-- Use the shovel to remove 50 balls each. 
-- Count the number of red balls and thus compute the proportion of the 50 balls that are red.
-- Return the balls into the bowl.
-- Mix the contents of the bowl a little to not let a previous group's results influence the next group's. 
+- Return the 50 balls used earlier back into the bowl and mix the contents of the bowl. This is done to ensure that each new sample is not influenced in any way by the previous sample.
+- Take a new sample with the shovel and obtain the sample proportion.
 
-```{r sampling-exercise-3b, echo=FALSE, fig.show='hold', fig.cap="Repeating sampling activity 33 times.", purl=FALSE, out.width = "30%"}
+```{r sampling-exercise-3b, echo=FALSE, fig.show='hold', fig.cap="Repeating sampling activity.", purl=FALSE, out.width = "30%"}
 # Need new picture
 include_graphics(c("images/sampling/balls/tactile_2_a.jpg", "images/sampling/balls/tactile_2_b.jpg", "images/sampling/balls/tactile_2_c.jpg"))
 ```
 
-Each of our 33 groups of friends make note of their proportion of red balls from their sample collected. Each group then marks their proportion of their 50 balls that were red in the appropriate bin in a hand-drawn histogram as seen in Figure \@ref(fig:sampling-exercise-4).
 
-```{r sampling-exercise-4, echo=FALSE, fig.cap="Constructing a histogram of proportions.", purl=FALSE, out.width = "80%"}
+Not surprisingly, different samples may produce different proportions of red balls. But these proportions follow some predictable patterns. A group of 33 students performed this activity previously and drew a histogram with these sample proportions as blocks. Figure \@ref(fig:sampling-exercise-4) shows students working on the histogram with two blocks drawn already representing proportion values and the third about to be added.
+
+```{r sampling-exercise-4, echo=FALSE, fig.cap="Students drawing a histogram of proportions.", purl=FALSE, out.width = "80%"}
 include_graphics("images/sampling/balls/tactile_3_a.jpg")
 ```
 
-Recall from Section \@ref(histograms) that histograms allow us to visualize the *distribution* \index{distribution} of a numerical variable. In particular, where the center of the values falls and how the values vary. A partially completed histogram of the first 10 out of 33 groups of friends' results can be seen in Figure \@ref(fig:sampling-exercise-5).
+Recall from Section \@ref(histograms) that histograms help us visualize the *distribution* \index{distribution} of a numerical variable. In particular, where the center of the values falls and how the values vary. A histogram of the first 10 sample proportions can be seen in Figure \@ref(fig:sampling-exercise-5).
```

Comment:
This is another spot where I hesitate to just use the phrase "sample proportions" here while we are building up the concepts. I want to make sure the cognitive load is not too much for students. If you have a good reason to switch it up beyond just brevity, I'm all ears.

Reply:
Got it! See my new edits.